csrankings.csv

```diff
@@ -11724,3 +11724,57 @@ Zygmunt J. Haas,University of Texas at Dallas,http://www.utdallas.edu/~zjh130030
+Ingrid Nunes,Universidade Federal do Rio Grande do Sul,http://inf.ufrgs.br/~ingridnunes,NOSCHOLARPAGE
+Jacob Scharcanski,Universidade Federal do Rio Grande do Sul,http://www.inf.ufrgs.br/~jacobs,o9csf6QAAAAJ
+João Luiz Dihl Comba,Universidade Federal do Rio Grande do Sul,http://www.inf.ufrgs.br/~comba,NOSCHOLARPAGE
+José Palazzo Moreira de Oliveira,Universidade Federal do Rio Grande do Sul,http://www.inf.ufrgs.br/~palazzo,8Xw1b6oAAAAJ
+Juergen Rochol,Universidade Federal do Rio Grande do Sul,http://inf.ufrgs.br/~juergen,NOSCHOLARPAGE
+Karin Becker,Universidade Federal do Rio Grande do Sul,http://inf.ufrgs.br/~kbecker,aZA2pGQAAAAJ
+Leandro Krug Wives,Universidade Federal do Rio Grande do Sul,http://www.inf.ufrgs.br/~wives,VV5p1EEAAAAJ
+Leila Ribeiro,Universidade Federal do Rio Grande do Sul,http://www.inf.ufrgs.br/~leila,gVbcXTQAAAAJ
+Lisandro Zambenedetti Granville,Universidade Federal do Rio Grande do Sul,http://www.inf.ufrgs.br/granville,v9JjkyYAAAAJ
+Lucas Mello Schnorr,Universidade Federal do Rio Grande do Sul,http://inf.ufrgs.br/~schnorr/,nPAkDeYAAAAJ
+Luciana Porcher Nedel,Universidade Federal do Rio Grande do Sul,http://www.inf.ufrgs.br/~nedel,NOSCHOLARPAGE
+Luciana S. Buriol,Universidade Federal do Rio Grande do Sul,http://www.inf.ufrgs.br/~buriol,5BoIwjYAAAAJ
+Luciano Paschoal Gaspary,Universidade Federal do Rio Grande do Sul,http://www.inf.ufrgs.br/~paschoal,SnT7z5wAAAAJ
+Lucinéia Heloisa Thom,Universidade Federal do Rio Grande do Sul,http://inf.ufrgs.br/~lucineia/,MTLCh6oAAAAJ
+Lucio Mauro Duarte,Universidade Federal do Rio Grande do Sul,http://inf.ufrgs.br/~lmduarte/,NOSCHOLARPAGE
+Luigi Carro,Universidade Federal do Rio Grande do Sul,http://www.inf.ufrgs.br/~carro,vS6vkY4AAAAJ
+Luís C. Lamb,Universidade Federal do Rio Grande do Sul,http://www.inf.ufrgs.br/~lamb,NOSCHOLARPAGE
+Manuel Menezes de Oliveira Neto,Universidade Federal do Rio Grande do Sul,http://inf.ufrgs.br/~oliveira,f2UrnmAAAAAJ
+Mara Abel,Universidade Federal do Rio Grande do Sul,http://inf.ufrgs.br/bdi,0yuOx18AAAAJ
+Marcelo Soares Pimenta,Universidade Federal do Rio Grande do Sul,http://inf.ufrgs.br/~mpimenta,NOSCHOLARPAGE
+Marcelo Walter,Universidade Federal do Rio Grande do Sul,http://www.inf.ufrgs.br/~mwalter/,7VfUEj0AAAAJ
+Márcio Dorn,Universidade Federal do Rio Grande do Sul,http://inf.ufrgs.br/~mdorn,gc0ljMIAAAAJ
+Marcus Ritt ,Universidade Federal do Rio Grande do Sul,http://inf.ufrgs.br/~mrpritt,NOSCHOLARPAGE
+Mariana Luderitz Kolberg,Universidade Federal do Rio Grande do Sul,http://lattes.cnpq.br/7133274936570173,NOSCHOLARPAGE
+Marinho P. Barcellos,Universidade Federal do Rio Grande do Sul,http://www.inf.ufrgs.br/~marinho/,NOSCHOLARPAGE
+Paolo Rech,Universidade Federal do Rio Grande do Sul,http://inf.ufrgs.br/~prech,1EKLRakAAAAJ
+Paulo Martins Engel,Universidade Federal do Rio Grande do Sul,http://inf.ufrgs.br/~engel,3yK2X-MAAAAJ
+Philippe Olivier Alexandre Navaux,Universidade Federal do Rio Grande do Sul,http://lattes.cnpq.br/5554254760869075,NOSCHOLARPAGE

```

Comment:
Same comment as above. Please point to an institutional web page.

Reply:
Also, the full name of the institution is too long for the UI and it will need to be abbreviated. Please propose a shorter name.

Reply:
Thanks! 
I am using now the official abbreviation and replaced the official CV by the institutional web page when a personal homepage is not available.